.github/workflows/balena-ci.yml

```
@@ -5,19 +5,22 @@ on:
       - master
 
 jobs:
-  build:
+  balena-cloud-build:
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: true
       matrix:
-        fail-fast: true
         fleets: [testbot-rig, testbot-rig-beta, testbot-personal]
```

```suggestion
        fleets: [balena/testbot-rig, balena/testbot-rig-beta, balena/testbot-personal]
```
Use fleet slugs to simplify adding new fleets from other orgs if needed.